### DISCUSSION TITLE
Runs in eclipse, but not a compiled jar

### DISCUSSION THREAD

**Discussion by Charles92011** [devopvoid/webrtc-java#60] [Q&A] [answered]
When I execute the project in eclipse, it runs fine and I can make connections.
But when I export it as an executable jar file it runs, but as soon as I attempt a connection I get the exception:
```
java.lang.NoClassDefFoundError: Could not initialize class dev.onvoid.webrtc.PeerConnectionFactory
```
I'm running on windows 10, the windows native library is being imported into my jar file

Any suggestions?

**Comment by Charles92011** [comment, marked as answer]
This turned out to be a misconfigured pom.xml
```
<dependency>
  <groupId>javax.websocket</groupId>
  <artifactId>javax.websocket-api</artifactId>
  <version>1.0</version>
  <!--<scope>provided</scope>-->
</dependency>
```

**Comment by kinsleykajiva** [comment]
that project link us not public

**Reply by Charles92011** [reply]
@kinsleykajiva yeah, I had to take the project down. I tried to delete this question, but there doesn't appear to be a way.
I created a similar project at: https://github.com/Charles92011/webrtc-java-experimental-server

**Reply by kinsleykajiva** [reply]
ok its fine thank you
